src/objection/row-exists-where.js

```diff
@@ -0,0 +1,41 @@
+const Boom = require('boom');
+const get = require('lodash.get');
+const merge = require('lodash/merge');
+const ValidationError = require('../ValidationError');
+
+module.exports = (Model, column, whereColumn, contextValuePath, message, constraintOptions) =>
+  function(value, validatorOptions) {
+    const options = merge(
+      {
+        convert: true,
+        return404: true,
+        fetchOptions: { eager: '', eagerOptions: {}, eagerAlgorithm: 'WhereInEagerAlgorithm' },
```

Comment:
I don't know what the use case is for including all the eager stuff in this validator - I think you could probably remove it and we can reevaluate adding it back if/when we need it?

Reply:
I'm not the most familiar with objection's api - is this letting you do `where` calls on relationships somehow?

Reply:
This is the equivalent of withRelated, but better because you can customize it and actually use joins in a supported way.